custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
```

Comment:
Consider using template literals for string interpolation to improve readability and maintainability.

Suggested change:
```javascript
 text: `Bringing exploratory mindset to technical leadership, with six years of experience in software development and computational research`,
``` 

For reference, see [MDN Web Docs on template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).